frontend/src/assets/css/styles.scss

```diff
@@ -14,6 +14,10 @@
   box-sizing: border-box;
 }
 
+html {
+  scroll-behavior: smooth;
+}
```

Comment:
Modifying `html` is considered a bad practice. Does this property have any other side-effects except making the "to-anchor" scroll smoother?

Reply:
I got it